tests/data/simpleFramerISD.json

```diff
@@ -0,0 +1,96 @@
+{
+    "boresight": [
+        0.0,
+        0.0,
+        1.0
+    ],
+    "ccd_center": [
+        7.5,
+        7.5
+    ],
+    "ephemeris_time": 100.0,
+    "focal_length": 500,
+    "focal_length_epsilon": 1.0,
+    "ifov": 6.0,
+    "model_name": "UsgsAstroFramePluginCSM",
+    "spacecraft_name": "TEST_CRAFT",
+    "instrument_id": "TEST_SENSOR",
+    "target_name": "TEST_BALL",
+    "pixel_pitch": 0.1,
+    "itrans_line": [
+        0.0,
+        0.0,
+        10
+    ],
+    "itrans_sample": [
+        0.0,
+        10,
+        0.0
+    ],
+    "transx": [
+        0.0,
+        0.1,
+        0.0
+    ],
+    "transy": [
+        0.0,
+        0.0,
+        0.1
+    ],
+    "min_elevation": -1,
+    "max_elevation": 1,
+    "nlines": 16,
+    "nsamples": 16,
+    "original_half_lines": 8.0,
+    "original_half_samples": 8.0,
+    "omega": 0,
+    "phi": 0,
+    "kappa": 0,
+    "semi_major_axis": 10,
+    "semi_minor_axis": 10,
+    "transx": [
+        0.0,
+        0.1,
+        0.0
+    ],
+    "transy": [
+        0.0,
+        0.0,
+        0.1
+    ],
+    "x_sensor_origin": 1000,
+    "y_sensor_origin": 0,
+    "z_sensor_origin": 0,
+    "x_sensor_velocity": 1,
+    "y_sensor_velocity": 0,
+    "z_sensor_velocity": 0,
+    "x_sun_position": 100,
```

Comment:
Were all of these keywords required by the framer plugin? (e.g. sun position, odt_x, etc.)?

Reply:
In UsgsAstroFramePlugin.cpp

```
const std::string UsgsAstroFramePlugin::_ISD_KEYWORD[] =
{
    "boresight",
    "ccd_center",
    "ephemeris_time",
    "focal_length",
    "focal_length_epsilon",
    "ifov",
    "instrument_id",
    "itrans_line",
    "itrans_sample",
    "kappa",
    "min_elevation",
    "max_elevation",
    "model_name",
    "nlines",
    "nsamples",
    "odt_x",
    "odt_y",
    "omega",
    "original_half_lines",
    "original_half_samples",
    "phi",
    "pixel_pitch",
    "semi_major_axis",
    "semi_minor_axis",
    "spacecraft_name",
    "starting_detector_line",
    "starting_detector_sample",
    "target_name",
    "transx",
    "transy",
    "x_sensor_origin",
    "y_sensor_origin",
    "z_sensor_origin",
    "x_sensor_velocity",
    "y_sensor_velocity",
    "z_sensor_velocity",
    "x_sun_position",
    "y_sun_position",
    "z_sun_position"
};
```

Reply:
`canModelBeConstructedFromISD` checks that all over those have non-empty values